### ISSUE TITLE
brownie style dependencies when compiling a `BrownieProject`

### ISSUE THREAD

**Issue by unparalleled-js** [ApeWorX/ape#799]
### Overview

New issue coming out of: https://github.com/ApeWorX/ape/issues/749

Dependencies in ape work differently than dependencies in Brownie.
In ape, the dependencies have to be manifest-able. In brownie, they just get pulled in.

If we allow dependencies for Brownie projects to work _brownie-style_, we can unblock a lot of user migrating over. And it makes sense, technically speaking!

**SIZE NOTE**: I am putting a 5 because of potential refactor complications but feel free to adjust 😅 

### Specification

* Don't make manifest from dependencies download when processing brownie dependencies.

NOTE: May require factoring config processing for brownie dependencies. Unknowns alert!

https://linear.app/apeworx/issue/APE-206/brownie-style-dependencies-when-compiling-a-brownieproject

### Dependencies

Include links to any open issues that must be resolved before this feature can be implemented.

**Comment by PatrickAlphaC** [comment]
Note: `ContractType collision across compiler plugins ` is what you'll get right now if you try to compile a project where contracts have the same name but different versions of solidity.

**Comment by hhamud** [comment]
Any plans on completing this issue?

**Comment by unparalleled-js** [comment]
> Any plans on completing this issue?

I am working on this now.
